pages/dkp/kommander/1.4/workspaces/workspace-platform-services/index.md

```diff
@@ -50,4 +51,7 @@ If desired, the Kubeaddons Controller can also be disabled, though it is highly
 | prometheusadapter                    | True                 | 1.16 - 1.20                   | True                                  |
 | reloader                             | True                 | 1.16 - 1.20                   | True                                  |
 | traefik                              | True                 | 1.16 - 1.20                   | True                                  |
-| traefik-forward-auth-kommander       | True                 | 1.16 - 1.20                   | False                                 |
+| traefik-forward-auth                 | True                 | 1.16 - 1.20                   | False                                 |
```

Comment:
This is the actual name of the addon that is federated so we probably want to leave this

Reply:
So, I'm going to push back a little, but I'm open to changing it. @gracedo - in the other pages and in the UI, it only says this: 
`traefik-forward-auth`
So, I think we should edit it to be the above, because in the UI it only says traefik-forward-auth, not traefik-forward-auth-kommander.